synth.py

```diff
@@ -21,19 +21,22 @@
 logging.basicConfig(level=logging.DEBUG)
 
 # run the gapic generator
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 versions = ['v1']
 for version in versions:
- library = gapic.node_library(
-   'cloudbuild',
-   config_path='/google/devtools/cloudbuild/artman_cloudbuild.yaml',
-   version=version)
- s.copy(library, excludes=['README.md', 'package.json'])
+  library = gapic.typescript_library(
+    'cloudbuild',
+    generator_args={
+      "grpc-service-config": "google/devtools/cloudbuild/v1/cloudbuild_grpc_service_config.json"
+    },
+    proto_path='/google/devtools/cloudbuild/v1',
+    version=version)
+s.copy(library, excludes=['README.md', 'package.json'])
 
 # Copy common templates
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library()
-s.copy(templates, excludes=[])
+s.copy(templates, excludes=['.jsdoc.js'])
```

Comment:
I don't think we want that.  You can pass a param to the common templates thingy with the src location (see what we do for nodejs-storage)